src/main/kotlin/com/openlattice/chronicle/notifications/Notification.kt

```diff
@@ -0,0 +1,23 @@
+package com.openlattice.chronicle.notifications
+
+import java.time.OffsetDateTime
+import java.util.*
+
+/**
+ * @author Todd Bergman <todd@openlattice.com>
+ */
+
+data class Notification(
+    val id: UUID,
+    val candidateId: UUID,
+    val organizationId: UUID,
+    val studyId: UUID,
+    val createdAt: OffsetDateTime = OffsetDateTime.now(),
+    var updatedAt: OffsetDateTime = OffsetDateTime.now(),
+    var status: String,
+    var messageId: String,
+    val type: String,
+    val body: String,
+    val email: String?,
```

Comment:
default to null (matches deserialization behavior) to make it easier to construct objects in kotlin